tests/integration/common-operator-integ-suite.sh

```diff
@@ -172,13 +177,33 @@ main_test() {
     ${COMMAND} get ns "${CONTROL_PLANE_NS}" >/dev/null 2>&1 || ${COMMAND} create namespace "${CONTROL_PLANE_NS}"
     sed -e "s/version:.*/version: ${ver}/g" "${ISTIO_MANIFEST}" | ${COMMAND} apply -f - -n "${CONTROL_PLANE_NS}"
 
+    echo "Wait for Istio to be Reconciled"
```

Comment:
Do we need this? Or just wait for `Ready` is enough?

Reply:
Although `Ready` should be enough, it makes sense to check `Reconciled`. Take this as an additional test assertion.